compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -4749,13 +4632,35 @@ public String toString() {
         FUNCTION_IN_DEFAULT_VALUE
     }
 
-    private static class PotentiallyInvalidAssignmentInfo {
-        List<Location> locations;
-        BLangInvokableNode enclInvokable;
-
-        private PotentiallyInvalidAssignmentInfo(List<Location> locations, BLangInvokableNode enclInvokable) {
-            this.locations = locations;
-            this.enclInvokable = enclInvokable;
-        }
+    /**
+     * @since 2.0.0
+     */
+    public static class AnalyzerData {
+        SymbolEnv env;
+        SymbolEnv newEnv;
+        BLangNode parent;
+        int loopCount;
+        boolean loopAlterNotAllowed;
+        boolean inInternallyDefinedBlockStmt;
+        int transactionCount;
+        boolean failureHandled;
+        boolean failVisited;
```

Comment:
shall we reorder the fields based on context. Like all the fields related to worker system in one group and divide with a comment.

Reply:
@KRVPerera, I have changed the order, can you check whether it looks okay?